enterprise/server/auth/auth.go

```diff
@@ -532,6 +536,19 @@ func (a *OpenIDAuthenticator) getAuthConfigForSlug(slug string) authenticator {
 	return nil
 }
 
+func (a *OpenIDAuthenticator) getAuthCodeOptions(r *http.Request) []oauth2.AuthCodeOption {
+	options := []oauth2.AuthCodeOption{}
+	if !a.env.GetConfigurator().GetDisableRefreshToken() {
+		options = append(options, oauth2.AccessTypeOffline)
+	}
+	sessionID := GetCookie(r, sessionIDCookie)
+	// If a session doesn't already exist, force a consent screen.
+	if sessionID == "" {
+		options = append(options, oauth2.ApprovalForce)
+	}
```

Comment:
is this necessary?

Reply:
Not strictly, but I find it useful for being able to switch between Google accounts if necessary. Kinda hate when you log, out and then try to re-login with another account and you get quickly auto-logged in without a consent screen.

Reply:
OK, wfm.